host/libvirt_lxc_backend.go

```diff
@@ -669,11 +672,28 @@ func (c *libvirtContainer) watch(ready chan<- error) error {
 			g.Log(grohl.Data{"at": "get_streams", "status": "error", "err": err.Error()})
 			return err
 		}
+
 		log := c.l.openLog(c.job.ID)
 		defer log.Close()
-		// TODO: log errors from these
-		go log.Follow(1, stdout)
-		go log.Follow(2, stderr)
+
+		muxConfig := logmux.Config{
+			AppName: c.job.Metadata["flynn-controller.app"],
```

Comment:
The `flynn` prefix is the host id, which should already be in the message somewhere? If not, it should be added.